src/ggrc/models/mixins/with_workflow_state.py

```diff
@@ -116,3 +121,43 @@ def get_workflow_state(cls, cycles):
           return cls.OVERDUE
       current_cycles.append(cycle_instance)
     return cls._get_state(current_cycles)
+
+
+class CycleTaskable(WithWorkflowState):
+  """CycleTaskable mixin."""
+
+  @declared_attr
+  def cycle_task_group_object_tasks(cls):  # pylint: disable=no-self-argument
+    """CycleTaskGroupObjectTasks to which object is mapped."""
+
+    secondary_join = """CycleTaskGroupObjectTask.id == case(
+                        [(Relationship.source_type == {},
```

Comment:
Lets put a placeholder for `source_type` values into `''`, shall we? The value here is a string so it seems it should be wrapped with `''`. Please correct me if I am wrong.
```suggestion
                        [(Relationship.source_type == '{}',
```

Reply:
@MikalaiMikalalai Overall changes look good to me, please address the previous comment though.

Reply:
Fixed. But I don't know if it's really needed.